src/site/content/en/blog/loading-ads-page-speed/index.md

```diff
@@ -0,0 +1,207 @@
+
+- Ad slots which are not initially visible can align with the content next to them, providing an opportunity for contextual relevance. This alignment can lead to higher user engagement as users discover ads that are related to the content they are exploring.
+- When designed thoughtfully, below the fold ads can seamlessly integrate with the surrounding content, appearing less disruptive to users. This integration—also known as [native advertising](https://support.google.com/admanager/answer/6366845?hl=en)—can result in a more harmonious user experience.
+- Scroll required ad placements offer more creative design and format flexibility, with ample space and freedom to experiment. Video ads, interactive elements, and larger images can be lazy loaded to capture user attention without disrupting their experience.
+
+However, the following considerations should be given for below the fold ad placements:
+
+- While below the fold ad placements can be effective, it's essential to ensure that users are encouraged to scroll down. Implementing visual cues or content teasers can entice users to explore further, increasing the likelihood of ad visibility.
+- The placements of ads below the fold should not compromise content quality or readability. Maintaining a balance between ads and content to avoid overwhelming users and to ensure positive user experience.
+- Unlike above the fold ad placements, below the fold ads may not need to load immediately. Delaying the loading of these ads until they are close to entering the user's viewport can help improve overall page load speed and reduce initial page rendering times.
+
+When used strategically, below the fold ads can complement above the fold ads, and provide a platform for creative ad formats and contextual relevance. However, optimizing visibility, balancing content, and managing ad load timing are key considerations to ensure a positive user experience. 
+
+Current Google Publisher Tag (GPT) best practices:
+
+- [General best practices](https://developers.google.com/publisher-tag/guides/general-best-practices)
+- [Ad Best Practices](https://developers.google.com/publisher-tag/guides/ad-best-practices)
+- [Minimize layout shift](https://developers.google.com/publisher-tag/guides/minimize-layout-shift)
+- [Monitor performance](https://developers.google.com/publisher-tag/guides/monitor-performance)
+- [Avoiding common implementation mistakes](https://developers.google.com/publisher-tag/common_implementation_mistakes)
+
+## Lazy load ads where appropriate
+
+<figure>
+  {% Img src="image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/o3fxVz2ih5esYzX5TjyU.png", alt="A visualization of lazy loading versus not lazy loading resources. When resources are lazy loaded, bandwidth is conserved during page load, and resources are deferred to the point at which the user is likeliest to see them.", width="800", height="450" %}
```

Comment:
This image references "Automatic lazy loading in lite mode". Lite mode doesn't do this anymore. Any way to edit this image to remove that?